vcpkg/portfile.cmake

```
@@ -0,0 +1,22 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Guekka/bsa
+    REF 459c83f4c155f088fa268e0bb502bfe7a9d90ffe
+    SHA512 bf01bb2f6661571856571bda421466412222a8c8bc95faa29767421e6e1d65babdde5d104fd276314737be535b023c39dddb88ecaea1079b64643ad386d51bad
+    HEAD_REF master
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+)
+vcpkg_install_cmake()
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/bsa)

```

```suggestion
vcpkg_cmake_configure(
	SOURCE_PATH "${SOURCE_PATH}"
	OPTIONS
		-DBUILD_TESTING=OFF
)
vcpkg_cmake_install()
vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/bsa")
```
These methods are deprecated